src/sentry/static/sentry/app/utils/cookies.jsx

```diff
@@ -0,0 +1,17 @@
+export default {
+  get: function (name) {
+    let cookieValue = null;
+    if (document.cookie && document.cookie !== '') {
+      let cookies = document.cookie.split(';');
+      for (let i = 0; i < cookies.length; i++) {
+        let cookie = jQuery.trim(cookies[i]);
```

Comment:
`String.prototype.trim` is [a thing now](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) – let's use that instead of jQuery.